deepchem/models/torch_models/tests/test_gan.py

```diff
@@ -0,0 +1,174 @@
+import deepchem as dc
+# import numpy as np
+import pytest
+# import tempfile
+from flaky import flaky
+
+# try:
+if True:
+    import torch
+    import torch.nn as nn
+    import torch.nn.functional as F
+
+    class Generator(nn.Module):
```

Comment:
Move this outside the try block

Reply:
done

Reply:
Just add a comment that since these are helper classes that depend on torch, they need to be in the try/catch block

Reply:
Done